src/Microsoft.ML.Recommend/Microsoft.ML.Recommend.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
```

Comment:
>Microsoft [](start = 52, length = 9)

Did anyone mention tests? No? Ok. WE NEED TESTS!

Reply:
Sure. I am working on IEstimator and ITransformer for matrix factorization, which is the first step of adding tests based on new APIs.

Reply:
Added two tests. One checks if IEstimator can be trained,  saved, and loaded. The other one checks if prediction matches the saved score.

---
In reply to: [225361720](https://github.com/dotnet/machinelearning/pull/1263#discussion_r225361720) [](ancestors = 225361720)